content/cloud_security.md

```diff
@@ -0,0 +1,214 @@
+
+- Customer data stored in the cloud
+- Platform, application, and identity & access management
+- Operating system, network, and firewall configuration
+- Data encryption, integrity, and authentication
+
+The **AWS Shared Responsibility Model** outlines how AWS commits to their cloud's security and where customers need to focus on security in the cloud.
+
+As a customer/company managing your own IT infrastructure (either bare metal or VMs) need to create a strategy and practice to manage all the elements depicted on the screen. AWS commits to the security of its platform and distributed infrastructure as a service to their customer. Their expertise and scale provide a massive benefit for customers, but you still need to have a plan to secure everything.
+
+{{< figure src="/images/inspec/cloud_security_aws_shared_model.png" title="Cloud Shared Responsibility Model" >}}
+
+#### Categories in Cloud Shared Responsibility Model
+
+There are three categories of responsibilities in the Shared Responsibility Model:
+
+- Inherited controls - Responsibilities that are always the providers.
+
+  The security responsibilities that are always the providers relate to safeguarding the infrastructure itself and access to, patching, and configuration of the physical hosts and the physical network on which the compute instances run, and where the storage and other resources reside. The security tasks extend by protecting end-user data, applications, operating systems, endpoints, and network traffic.
+
+- Customer-specific infrastructure - Responsibilities that are always the customer’s.
+
+  The security responsibilities that are always the customer’s include managing users and their access privileges (identity and access management), safeguarding cloud accounts from unauthorized access, the encryption and protection of cloud-based data assets, security of the zone data, and managing its security posture compliance.
+
+- Shared controls - Responsibilities that vary depending on the service model.
+
+  - Infrastructure-as-a-Service (IaaS)): A hybrid approach where organizations can manage some of their data and applications on-premise while relying on cloud providers to manage servers, hardware, networking, virtualization, and storage needs.
+
+  - Platform-as-a-Service (PaaS): Provides organizations the ability to streamline their application development and delivery by providing a custom application framework that automatically manages operating systems, software updates, storage, and supporting infrastructure in the cloud.
+
+  - Software-as-a-Service (SaaS): Cloud-based software hosted online and typically available on a subscription basis. Third-party providers manage all potential technical issues, such as data, middleware, servers, and storage, minimizing IT resource expenditures and streamlining maintenance and support functions.
```

Comment:
These shouldn't be hyphenated. See: https://developers.google.com/style/word-list#letter-i

Reply:
Ok - I have removed them. Actually, MSTP and CMS say to do.

Reply:
Would you please merge it?